src/sdk.js

```diff
@@ -279,3 +306,25 @@ Smartcar.AccessDenied = class extends Error {
     this.name = 'AccessDenied';
   }
 };
+
+/**
+ * Vehicle incompatible error returned by authorization flow.
```

Comment:
thoughts on adding a second sentence: "Will optionally have a `vehicleInfo` object if the user chooses to give permissions to provide that information. See our Connect flow documentation for more details."

Reply:
Great idea! Fixed